JavaScript/Topics/7.Promises/README.md

```diff
@@ -0,0 +1,6 @@
+# Promises In Javascript
+
+### Vedio Lecture On Promises
+Vedio-[promises](https://drive.google.com/file/d/1XRxoC3t9B_y9gARvmr95w5BfOz3-3PIf/view?usp=sharing)
```

Comment:
Please spell `Video` correctly. The rest seems OK!

Reply:
Oh I am really sorry @paramsiddharth

Reply:
It's OK, Devyank. :) @2devyank You can update your PR and we will approve and merge it.